seeding/create-log-records.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# If you're on Windows, run bash create-log-records.sh -w
+# Otherwise, run bash create-log-records.sh
+
+# NOTE: 
+# this script will fail if a user with $EMPLOYEE_ID as an id does not exist
+# it will also fail if three residents with ids 1, 2, and 3 do not exist
+
+# Import common functions
+source ./seed.sh
+
+# Run SQL script in docker container
+run_sql_script "
+INSERT INTO log_records (
+  employee_id, 
+  resident_id,
+  datetime, 
+  flagged, 
+  attn_to, 
+  note, 
+  tags, 
+  building 
+) 
+SELECT
+  $EMPLOYEE_ID,
+  (ARRAY(SELECT generate_series(1, $RESIDENT_ROWS)))[floor(random() * $RESIDENT_ROWS + 1)], 
```

Comment:
This works, but I could see this becoming annoying when for ex. 
- Insert 25 residents, they have ID's 1 through 25
- Delete them
- Insert 25 more residents, they now have ID's 26 through 50
- There's 25 resident rows, but now the ID's are out of range

Kind of an edge case to be honest, but there's probably less room for error to tie it to a specific value. It would be even better if we could get the current IDs from the DB.

If we're going this route, I think it would be nice to expand this to EMPLOYEE_ID and ATTN_TO_ID as well

Reply:
yeah you're right :( I'm thinking the auto_increment can be reset in the script before running the insert again